omod/src/main/webapp/resources/scripts/components/LabTrackingOrderDetailsController.js

```diff
@@ -158,6 +158,9 @@ angular.module("labTrackingOrderDetailsController", [])
               $scope.downloadPdf = function () {
                 LabTrackingDataService.downloadPdf($scope.order);
               };
+              $scope.isObject = function (obj) {
+                return typeof obj === 'object';
+              }
```

Comment:
Might have just been easier to have a "formatDiagnosis" function here, as you are re-using the isObject method above, but then copying the same code snippet that uses that to format the diagnosis.

Reply:
Agreed... I changed this.  The bigger nit (that has't been resolved) is that I repeated this function three times because I didn't remember off the top of my head how to globally define a filter...